bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/extensions/ExtensionResource.java

```diff
@@ -138,7 +138,7 @@ public Response getTypes(@HeaderParam("Accept-Language") @ApiParam(value = "lang
     }
 
     @GET
-    @Path("/{extensionId: [a-zA-Z_0-9-]+}")
+    @Path("/{extensionId: [a-zA-Z_0-9-:]+}")
```

Comment:
I am a fan of streamlining it but not sure if we should add the colon here or rather remove it from install/uninstall methods.

@kaikreuzer You introduced the colon in https://github.com/eclipse-archived/smarthome/pull/3076 because the marketplace adds `marcetplace:` prefix. As the Marketplace itself is EOL. What is your opinion on this?

Reply:
> As the Marketplace itself is EOL

I would not say so. What we will drop is the [Eclipse IoT Marketplace extension service](https://github.com/openhab/openhab-addons/tree/2.5.x/bundles/org.openhab.extensionservice.marketplace), but not the general marketplace extension mechanism (see https://github.com/openhab/openhab-addons/issues/6673).
We will therefore need a way to differentiate between "distro" add-ons and stuff that comes from other (external) sources. And using colons for such namespaces imho still makes sense, so I would keep it in place.